src/main/scala/md/fusionworks/adam/jbrowse/models/convert/Convert.scala

```diff
@@ -0,0 +1,61 @@
+package md.fusionworks.adam.jbrowse.models.convert
+import org.apache.spark.{SparkConf, SparkContext}
+import org.bdgenomics.adam.rdd.{ADAMContext, ADAMRDDFunctions}
+
+class Convert {
+
+  def fasta(link: String){
```

Comment:
fastaToParquet or concrete type insted of parquet 
in method parameters put inputPath and outputPath